_docs/dependency-injection.md

```diff
@@ -160,6 +161,6 @@ class AStatefulSessionBean
 }
 ```
 
-In that example, the container injects an instance of `AStatelessSessionBean` at runtime by invoking the `injectAStatelessSessionBean` method passing the instance as argument.
+In the above example, the container injects an instance of `AStatelessSessionBean` at runtime by invoking the `injectAStatelessSessionBean` method passing the instance as an argument.
 
-> Method injection only works on methods that have exactly one argument. As described above, the container does not inject a real instance of the bean. Instead, it injects a proxy. That proxy not extends the class or, if given, implements the interfaces of your bean. So, do NOT type hint the argument with the class or an interface name.
+> Method injection only works on methods that have exactly one argument. As described above, the container does not inject a real instance of the bean. Instead, it injects a proxy. That proxy extends the class or, if given, implements the interfaces of your bean. So, do NOT type hint the argument with the class or an interface name.
```

Comment:
This is way above my lack of knowledge. I've changed the sentence. But, please do make sure it is correct.

Scott